meta-balena-common/recipes-core/plymouth/plymouth_%.bbappend

```diff
@@ -3,10 +3,6 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 inherit deploy
 
 SRC_URI_append = " \
-    file://0001-plymouth-systemd-append.patch \
```

Comment:
NIT: I would reword a little the summary. This commit doesn't update plymouth per se.

Reply:
Reworded commit message